src/components/ModalEvent/ModalEvent.jsx

```diff
@@ -39,12 +45,23 @@ const ModalEvent = ({ event: eventValue, onCancel, onSubmit }) => {
   const handleDayChange = (value) => setEvent({ ...event, day: value });
   const handleTimeChange = (value) => setEvent({ ...event, time: value });
 
-  const isValid = Object.values(event).every(Boolean);
-  const isEventConflict = () => userEvent[userName]?.[day]?.[time] ? alert('The event already exists. Change the date or edit the current event from the calendar.') : onSubmit(event)
+  const isValid = Object.values(event).every((value) => {
+    return Boolean(value);
+  });
+
+  const handleClickConfirm = () => {
+    const existValue = userEvent[userName]?.[day]?.[time]
+    if(existValue && isNewEvent) {
+      alert('The event already exists. Change the date or edit the current event from the calendar.')
+    } else {
+      onSubmit(event)
+    }
+  }
 
   const renderModalFooter = () => (<>
     <Button className="modal-event__control" block label="Cancel" onClick={ onCancel }/>
-    <Button className="modal-event__control" block label="Confirm" disabled={ !isValid } onClick={ isEventConflict }/>
+    { !isNewEvent && isEditMode && <Button className="modal-event__control" block label="Delete" disabled={ !isValid } onClick={ () => onSubmit({subject: '', userName: event.userName, day: event.day, time: event.time}) }/>}
```

Comment:
please turn the code into multiple lines
```
{
  !isNewEvent && isEditMode && <Button className="modal-event__control"
    block
    label="Delete"
    ...
    onClick={ () => ... } />
  
}
```

Reply:
Done